dist/README.md

```diff
@@ -33,6 +33,19 @@ or the un-minified version as:
 
 You can grab the relevant MathJax files in `./dist/extras/mathjax/`.
 
+By default, plotly.js will modify the global MathJax configuration on load.
```

Comment:
`dist/README.md` is a generated file, you'll need to move this block to 

https://github.com/plotly/plotly.js/blob/master/tasks/stats.js

Reply:
Done in 57e32ba